### PR TITLE
finished the last 3 steps

### PR DESCRIPTION
**Pull request by OlgaCornovan**
Finished the last 3 steps of the test case

### CODE REVIEW

File: src/main/java/steps/AESteps.java [hunk 1]
```diff
@@ -25,6 +26,22 @@ public void iFillInInvalidEmail (String word) {
         calcPage.fillInInvalidEmail(word);
     }
 
+    @And("^I fill in with my valid password \"([^\"]*)\"$")
+    public void IFillInValidPassword (String word) {
```
Comment (Zibokk) [inline src/main/java/steps/AESteps.java]: Please use unic String. You have been using word 2 times `String word`

File: src/test/resources/A&EWatches.feature [hunk 2]
```diff
@@ -8,4 +8,6 @@
 #      And I click login amazon button
```
Comment (Zibokk) [inline src/test/resources/A&EWatches.feature:8]: After you'll create Amazon steps delete comments

File: src/main/java/steps/AESteps.java (hunk 1 above)
Comment (OlgaCornovan) [reply src/main/java/steps/AESteps.java]: Found specific variables for the String
`public void iVerifyErrorMessage (String message){
        calcPage.verifyErrorMessage(message); is `

` public void IFillInValidPassword (String password) {
        calcPage.fillInValidPassword(password);`

`public void iFillInInvalidEmail (String email) {
        calcPage.fillInInvalidEmail(email);`

File: src/test/resources/A&EWatches.feature (hunk 2 above)
Comment (OlgaCornovan) [reply src/test/resources/A&EWatches.feature:8]: I'll change when I'll create Amazon test cases

**Pull request merged**